.github/workflows/zfs-qemu.yml

```
@@ -6,15 +6,48 @@ on:
 
 jobs:
 
+  test-config:
+    name: Generate configuration for QEMU test
```

```suggestion
    name: Setup
```

How about renaming this to simply "Setup" to be more generic and symmetric with the final "Cleanup" job.